tests/BaseTestCase.php

```diff
@@ -46,7 +46,7 @@
      */
     public function setUp()
     {
-        Models::setUsersModel(User::class);
+        Bouncer::useUserModel(User::class);
```

Comment:
Why did you change this?

Also, since those methods on the Bouncer class will now be non-static, we can't even use `Bouncer` here.

Reply:
Hello,
I changed this because there is the method on the Bouncer class that wraps up the code, and to make sure the tests fail in case there is an error like it was with returning $this in a static context.
As you stated the error was that the method shouldn't be static, not that it returned $this, so now I understand it, and it`s not correct in the new context.